backend/app/middlewares/loginAdminUser.js

```diff
@@ -0,0 +1,33 @@
+const authJwt = require("./authJwt");
+
+isAdmin = (req, res, next) => {
+  User.findById(req.userId).exec((err, user) => {
+    if (err) {
+      res.status(500).send({ message: err });
+      return;
+    }
+
+    Role.find(
+      {
+        _id: { $in: user.roles }
+      },
+      (err, roles) => {
+        if (err) {
+          res.status(500).send({ message: err });
+          return;
+        }
+
+        for (let i = 0; i < roles.length; i++) {
+          if (roles[i].name !== "admin") {
```

Comment:
The current code `roles[i].name !== "admin"` means if any of the roles is not an admin role we would return 403 but that's not the desired effect. Instead we should check `roles[i].name === "admin"` and then we call the `next()` function and return.